guides/Getting-Started.md

```diff
@@ -258,15 +260,41 @@ Spago can be used to turn our PureScript code into JavaScript suitable for use i
     Build succeeded.
     Bundle succeeded and output file to index.js
 
-All the code in the `src` directory and any project dependencies have been compiled to JavaScript. The resulting code is bundled as `index.js` and has also had any unused code removed, a process known as dead code elimination. This `index.js` file can now be included in an HTML document. If you try this, you should see the words "Hello, World!" printed to your browser's console.
+All the code in the `src` directory and any project dependencies have been compiled to JavaScript. The resulting code is bundled as `index.js` and has also had any unused code removed, a process known as dead code elimination. This `index.js` file can now be included in an HTML document.  Try this by creating the following `index.html` file in your `my-project` directory:
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Euler Exercise</title>
+</head>
+
+<body>
+  <div id="container"></div>
+  <script src="./index.js"></script>
+</body>
+
+</html>
+```
+
+See this HTML file work as follows:
+
+1.  Run `npx parcel index.html --open` in a terminal opened to your `my-project`.
```

Comment:
I'd rather not involve npx or parcel at this stage. Why not ask the reader to open index.html in a browser directly (i.e. with a file:// url)? Some projects need to use an HTTP server for local development because of things like cookies, but that shouldn't be necessary here.

Reply:
@hdgarrood Without a server, opening the debugger console doesn't show the "The answer is 233168" line (at least in my system).  But you have more important concerns below.

Reply:
That's weird, it does for me. Does your index.js end with `PS["Main"].main();`? What browser are you using?

Reply:
@hdgarrood my bad; it worked after I asked my FireFox browser to "show content messages."

Reply:
I agree with @hdgarrood -- since we can just open this in the browser via the `index.html` file we may as well only require that vs. introducing a particular JavaScript bundler. I think we should only introduce the JS tools when they become necessary as they're outside our ecosystem.